### ISSUE TITLE
Aerosol optical depth in RRTMG

### ISSUE THREAD

**Issue by remcoverzijlbergh** [brian-rose/climlab#135]
Dear Brian,

When I was trying to give a different value of aerosol optical depth to RRTMG, I noticed that line 175 in rrtmg.py reads:
```
                     tauaer = tauc_sw,
```
This seems like a typo (the cloud optical depth is used instead of the aerosol optical depth). It should probably read:
```
                     tauaer = tauaer_sw,
```

I hope this is the correct way of filing this 'bug report'. Climlab is great, by the way. 

Best regards,

Remco

**Comment by brian-rose** [comment]
Hello @remcoverzijlbergh,

I believe you are correct! I don't think we have any tests that touch the aerosol arguments, so nobody has ever caught that before. I will fix.

**Comment by brian-rose** [comment]
This is fixed now in climlab 0.7.9.
Thanks for the report!

**Comment by remcoverzijlbergh** [comment]
Great! 

Keep up the good work!
